tests/test_run_reference_vmm.py

```diff
@@ -370,3 +406,32 @@ def test_reference_vmm_mem(kernel):
         expect_mem(vmm_process, expected_mem_mib)
 
         shutdown(vmm_process)
+
+
+@pytest.mark.parametrize("kernel", KERNELS_INITRAMFS)
+def test_serial_console_piping(kernel, tmp_path):
```

Comment:
What is `tmp_path`? Is it some sort of keyword that gives you a temporary directory?